htlcswitch/mailbox_test.go

```diff
@@ -20,12 +22,8 @@ func TestMailBoxCouriers(t *testing.T) {
 
 	// First, we'll create new instance of the current default mailbox
 	// type.
-	mailBox := newMemoryMailBox(&mailBoxConfig{
-		clock:  clock.NewDefaultClock(),
-		expiry: time.Minute,
-	})
-	mailBox.Start()
-	defer mailBox.Stop()
+	ctx := newMailboxContext(t, time.Now(), testExpiry)
```

Comment:
I'd recommend using `clock.TestClock` instead of real time here.

Reply:
`newMailboxContext` already creates a `TestClock` internally ;)

Reply:
Sneaky :) Somehow `time.Now()` made me think it does depend on actual time. Perhaps it's a good idea to fix the time anyway as then when we grep for `time.Now()` in tests we won't find this one. (Not a biggie though)

Reply:
> we grep for time.Now()

not sure i follow, why wouldn't this line be found? do you mean `clock.Now()`?

Reply:
There are a bunch of tests currently depending on `time.Now()` which is probably wrong and in the long run should be fixed to use `TestClock` instead. 
Just made a quick ` ag "time.Now()" | grep "_test.go" | wc -l`, seems like we have 144 occurrences.